emma_cornelius_folder/stocks challenge.py

```diff
@@ -0,0 +1,65 @@
+print("Challenge 3.2: Playing with the stock market")
+
+print()
+
+# Creating variables to store the current (approximate) market price of these 5 companies - Amazon, Apple, Facebook, Google and Microsoft.
+
+amazon = 3000
+apple = 100
+fb = 250
+google = 1400
+msft = 200
+
+
+print("Challenge 3.2.1: Taking user input")
+# TODO: Write code to ask the client his name and save it to a variable.
+name = input("What is your name?")
+# TODO: Write code to ask the client his savings and save it to another variable.
+savings = int(input("What is your savings?)"))
+# TODO: Write code to ask the client the stock he is interested in and save it to another variable, as shown below.
+stock = input("Which stock are you interested in? Type 'amzn' for Amazon, 'aapl' for Apple, 'fb' for Facebook, 'goog' for Google and 'msft' for Microsoft.")
+print()
+
+print("Challenge 3.2.2: Perform user-specific calculations")
+# TODO: You have all 3 user inputs stores in variables. Based on that, write conditional (if-elif-else) statements to find out the number of stocks of the company that can be purchased with the savings amount.
+numStocks = 0
+if stock == "aapl":
+    price = apple
+    numStocks = savings/price
+elif stock == "amzn":
+    price = amazon
+    numStocks = savings/price
+elif stock == "fb":
+    price - fb
+    numStocks = savings/price
+elif stock == "goog":
+    price = google
+    numStocks = savings/price
+elif stock == "msft":
+    price = msft
+    numStocks = savings/price
+else: 
+    price == "NA" 
+
+
+'''
+Your code should look like this:
+if stock == "amzn":
+    ...
+elif ...
+else ...
+'''
+
+print()
+
+print("Challenge 3.2.3: Output for the user the result")
+# TODO: COnce you have calculated the number of stocks that can be purchased, print the result for the client. Result should be in a format like this:
+
+    
+# Alex has $5000 in savings and he can buy 50 shares of Apple at the current price of $100.
+
+name = 'Alex'
+if price == "NA":
+    print("incorrect stock.")
+else:
+    print(f"{name} has ${savings} in savings and he can buy {numStocks} shares of {stock} at the current price of ${100}. ")
```

Comment:
current price of should be the cost of the stock. The example: `Alex has $5000 in savings and he can buy 50 shares of Apple at the current price of $100` uses $100 because that is the cost of the apple stock.

If the user selects another stock, it should show the price of that. Look back at your if/elif/else statements to find which variable you should use.